easier_net/sier_net.py

```diff
@@ -236,11 +243,16 @@ def fit(
         self,
         x: np.ndarray,
         y: np.ndarray,
-        max_iters: int = 100,
-        max_prox_iters: int = 100,
-        state_dict: dict = None,
-    ) -> None:
+    ):
         # Assemble data
+        self.net = SierNet(
+        input_filter_layer=self.input_filter_layer,
```

Comment:
fix indentation. btw, you could consider using the `black` python package for that.